components/form_1/form_1_fields.tsx

```diff
@@ -0,0 +1,94 @@
+import { FormikProps } from 'formik';
+import { ChangeEvent } from 'react';
+import style from '../../styles/home.module.scss';
+import env from '../../env/env.json';
```

Comment:
env.json, name & use of this file is not right, env files usually used for keeping the secrets like KEYS, PASSWORD, DB details etc but not for config files, maybe you want to have something like constant.js or config.js

Reply:
ok sure I will change the name